docs/migration/migrate_7_3.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[breaking-changes-7.3]]
+== Breaking changes in 7.3
+++++
+<titleabbrev>7.3</titleabbrev>
+++++
+
+This section discusses the changes that you need to be aware of when migrating
+your application to Kibana 7.3.
+
+See also <<release-highlights>> and <<release-notes>>.
+
+* <<breaking_73_dashboard_import_export>>
+
+//NOTE: The notable-breaking-changes tagged regions are re-used in the
+//Installation and Upgrade Guide
+
+[float]
+[[breaking_73_dashboard_import_export]]
+=== Dashboards Import/Export API
```

Comment:
To match the [other documentation](http://localhost:8000/guide/dashboard-import-api.html), I would change this to `Dashboard import and export APIs`.